packages/hop-node/src/watchers/SyncWatcher.ts

```diff
@@ -1262,6 +1390,16 @@ class SyncWatcher extends BaseWatcher {
     return true
   }
 
+  getIsRelayable = (
+    relayerFee: BigNumber
+  ): boolean => {
+    if (relayerFee.eq(0)) {
```

Comment:
since the condition is already a boolean it can be simplified with `return relayerFee.gt(0)`